src/style.css

```diff
@@ -1,3 +1,66 @@
-body {
+body,
+.nav {
   margin: 0;
+  padding: 0;
+  max-width: 1200px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  padding-right: 50px;
+  padding-left: 50px;
+}
+
+.nav {
+  justify-content: space-between;
+  display: flex;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.nav_link {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
```

Comment:
You can remove it, because `normal` is default value
```suggestion
```

Reply:
![man-mechanic_1f468-200d-1f527](https://github.com/mate-academy/layout_moyo-header/assets/129754902/c6e1610f-442a-448f-87a1-7a0432439acd)